avocado/core/loader.py

```diff
@@ -658,22 +658,25 @@ def _find_avocado_tests(self, path, class_name=None):
                 docstring = ast.get_docstring(statement)
                 # Looking for a class that has in the docstring either
                 # ":avocado: enable" or ":avocado: disable
-                if (safeloader.check_docstring_directive(docstring, 'disable')
-                   and class_name is None):
+                has_disable = safeloader.check_docstring_directive(docstring,
```

Comment:
I used to insist on this, but then the majority decided to ignore this warning. You have my full support to send PR like this accommodated with commit which enables this check in travis, otherwise this fix is needed.